config/locales/idv/en.yml

```diff
@@ -39,16 +39,16 @@ en:
         Please check that the personal and financial information you provided is accurate.
         To learn more about the information we need from you and how it’s used to verify your identity,
         please see our troubleshooting page.
-      activated: >
+      activated_html: >
         Your identity has been verified. If you need to change your verified information,
-        please %{link}.
+        please <a href=%{url}>%{link_text}</a>.
```

Comment:
What you are proposing is what I had to change in order to fix the `OutputSafety` offenses. In order for the link to appear properly in the view, you need to call `.html_safe`, which can be a security concern and is generally not recommended to use. Instead, it is recommended to use built-in helpers such as `content_tag` or the `_html` suffix in locales, which Rails automatically safely unescapes:
http://guides.rubyonrails.org/i18n.html#using-safe-html-translations

I'm open to another way to do this that doesn't result in a Rubocop offense.